retrofit/src/main/java/retrofit/Call.java

```diff
@@ -33,4 +33,5 @@
   void enqueue(Callback<T> callback);
   void cancel();
   Call<T> clone();
+  String infoForException();
```

Comment:
I really don't like this name. Is there any reason this shouldn't just be `toString()`? Otherwise I would call this `describe()`.